korgan/pretrial.py

```
@@ -177,7 +186,36 @@ def _today() -> str:
 
+def _body_paragraph(doc: Document, text: str) -> None:
+    value = str(text or "").strip()
+    if not value:
+        return
+    p = doc.add_paragraph(value)
+    p.alignment = WD_ALIGN_PARAGRAPH.JUSTIFY
+    p.paragraph_format.first_line_indent = Cm(1.25)
+    p.paragraph_format.space_after = Pt(6)
+
+
+def _deadline_paragraph(value: str, kk: bool) -> str:
```

Ruff (0.16.2)

[warning] Boolean-typed positional argument in function definition

(FBT001)